nmrfx-analyst-gui/src/main/java/org/nmrfx/processor/gui/RightSideBarContentUtility.java

```diff
@@ -0,0 +1,8 @@
+package org.nmrfx.processor.gui;
+
+public class RightSideBarContentUtility {
+    private RightSideBarContentUtility() {
+        throw new IllegalAccessError("Utiltity class cannot be instantiated");
+    }
+    public static final double MINIMUM_WIDTH = 400;
```

Comment:
There seemed to be 3 options to get the right pane sizing to work, 
a) set the minwidth in all 3 fxml files, 
b) set it programatically storing the value in a utility class (chose this option) or 
c) create a css class and set it there. 
Was trying to avoid having to update 3 files if we need to change the minimum width in the future. Not sure if one of the other options would be better or there is something else that is better?